### PR TITLE
Sprint 7.1 changes

### PR DESCRIPTION
**Pull request by clindeman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,4 +26,4 @@ This tool currently only supports ADCIRC depth grids.
 
 Issues can be reported through the repository on Github (https://github.com/nhrap-dev/FHIT)
 
-For questions contact hazus-support@riskmapcds.com
+For questions contact fema-hazus-support@fema.dhs.gov
```
Comment (be-cunning) [inline README.md:78]: Thanks for updating this...I think the other tools may need this as well...I'll check on that.

File: src/environment.yaml [hunk 2]
```diff
@@ -70,7 +70,7 @@ dependencies:
   - libspatialite=5.0.0=h37d8b57_0
   - libssh2=1.9.0=hb06d900_5
   - libtiff=4.2.0=hc10be44_0
-  - libwebp-base=1.2.0=h8ffe710_0
+  - libwebp-base=1.2.0=h8ffe710_2
```
[the remark this answers is not in the document]
Comment (be-cunning) [reply src/environment.yaml:75]: **TODO**: add **_boto3=1.18.47=pyhd8ed1ab_0_** to the list of dependencies.

File: Python_env/ADCIRC.py [hunk 3]
```diff
@@ -1,56 +1,187 @@
-'''2021 colin lindeman clindeman@niyamit.com colin.lindeman@hawaii.edu
+'''2021 colin lindeman clindeman@niyamit.com, colin.lindeman@gmail.edu
```
Comment (be-cunning) [inline Python_env/ADCIRC.py]: why is _**gmail**_ being used here, and not the **_niyamit.com_** email?

File: Flood-Hazard-Import-Tool.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+from src.manage import Manage
+
+if __name__=='__main__':
+    manage = Manage()
+    app_path = r'Python_env\gui_program.py'
+    try:
+        manage.checkForUpdates()
+        manage.startApp(app_path)
+    except Exception as e:
+        print(e)
```
Comment (be-cunning) [inline Flood-Hazard-Import-Tool.py:10]: This part will need some refactoring with the recent hazpy integration updates, but it's fine for now - just making a note.

**Review by be-cunning — DISMISSED** [review]

File: Python_env/views.py [hunk 5]
```diff
@@ -0,0 +1,558 @@
...
+
+import tkinter as tk
+import tkinter.ttk as ttk
+import os
+import time
+import json
+from pathlib import Path
+from threading import Thread
+import ctypes
+import adcirc
```
Comment (be-cunning) [inline Python_env/views.py:24]: adcirc needs to be capitalized in order to import correctly.

TODO: change **_adcirc_** to _**ADCIRC**_

File: Python_env/fhitsupport.py [hunk 6]
```diff
@@ -0,0 +1,111 @@
+'''2021 colin lindeman clindeman@niyamit.com, colin.lindeman@gmail.edu
```
Comment (be-cunning) [inline Python_env/fhitsupport.py]: Why is the gmail address in here?

File: Python_env/gui_program.py [hunk 7]
```diff
@@ -1,303 +1,26 @@
...
--prompt user to overwrite or not?
--give user feedback/status that the app is doing something, like getting list of storms or downloaded data
--give user feedback that file was downloaded successfully or not
--show just the file name and not the key/path at step 5
--adjust combobox widths to account for longest possible values, static or dynamic
--adcirc made changes to their folder structure around 2/16/21 which affected this gui
-'''
+"""clindeman
+python 3
+"""
```
Comment (be-cunning) [inline Python_env/gui_program.py:3]: Couldn't fit this comment anywhere else, but why does **Line 546** have the '**TODO Depth Unit**' in the combo box of values?

File: Python_env/views.py [hunk 8]
```diff
@@ -0,0 +1,558 @@
...
+        self.floodDepthGridSelection = tk.StringVar()
+        #self.fileKey = tk.StringVar() #TODO for download
+
+        self.yearList = ['Choose a Year...']
+        self.weathertypeList = ["Synoptic", "Tropical"]
+        self.stormList = ['Choose a Storm...'] 
+        self.advisoryList = ['Choose an Advisory...']
+
+        #TODO handle if website down or data format changed significantly
+        self.adcircData = adcirc.ADCIRC()
```
Comment (be-cunning) [inline Python_env/views.py:225]: adcirc needs to be capitalized in order to import correctly.

TODO: change adcirc to ADCIRC

File: docs/ADCIRC.txt [hunk 9]
```diff
@@ -0,0 +1,17 @@
+https://hazus.s3.us-east-2.amazonaws.com/index.html
+GUI like windows explorer. Source data does not have structure or links to files
+
+https://hazus.s3.us-east-2.amazonaws.com/ 
+XML file that appears to contain elements with file paths
+This entry makes me think it will only list 1000 files on one page <MaxKeys>1000</MaxKeys>
```
Comment (be-cunning) [inline docs/ADCIRC.txt]: Is this a limitation with the tool/the AWS API? If so, is there a way to get around it?

File: src/environment.yaml [hunk 10]
```diff
@@ -48,6 +49,7 @@ dependencies:
   - idna=2.10=pyh9f0ad1d_0
   - intel-openmp=2020.3=h57928b3_311
   - jenkspy=0.1.4=py39h4cdbadb_1002
+  - jmespath=0.10.0=pyh9f0ad1d_0
```
Comment (be-cunning) [inline src/environment.yaml:51]: I couldn't place this comment further up, but do we still need the _**hazpy**_ dependency on line 44?

File: README.md [hunk 11]
```diff
@@ -1,29 +1,78 @@
...
+**2021-Tropical-al19-30** = 19th storm in the Atlantic in 2021, advisory 30
+
+Avisory **00** is the hindcast data
+
 ## Contact
 
 Issues can be reported through the repository on Github (https://github.com/nhrap-dev/FHIT)
 
-For questions contact hazus-support@riskmapcds.com
+For questions contact fema-hazus-support@fema.dhs.gov
```
Comment (be-cunning) [inline README.md]: Thanks for updating this. This also reminds me that I need to change, and possibly edit, this in the other tools.

File: Python_env/__init__.py [hunk 12]
```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+"""
+    FHIT 
+    ~~~~~
+    FEMA tool to import flood hazard data into Hazus.
+
+    :copyright: © 2021 by FEMA's Natural Hazards and Risk Assesment Program.
+    :license: cc, see LICENSE for more details.
+    :author: Colin Lindeman; colinlindeman@hotmail.com, Colin Lindeman; clindeman@niyamit.com
```
Comment (be-cunning) [inline Python_env/__init__.py]: why is the **_hotmail_** account listed here?

File: Python_env/TropicalStormNames/readme.md [hunk 13]
```diff
@@ -0,0 +1,13 @@
+Atlantic, Eastern North Pacific and Central North Pacific Storm Name list json files created on 6/17/21 from the following websites:
```
Comment (be-cunning) [inline Python_env/TropicalStormNames/readme.md:1]: This is a minor nit: readme files are generally capitalized.

**Optional**: rename file from '**_readme_**' to '**_README_**'

File: Python_env/TropicalStormNames/easternnorthpacific.json [hunk 14]
```diff
@@ -0,0 +1,184 @@
+{
```
Comment (be-cunning) [inline Python_env/TropicalStormNames/easternnorthpacific.json:1]: **General question:** will the JSON lists be/need updated in the future? If so, how will they get updated? If so, is there a way to automate the process?

**Review by be-cunning — DISMISSED** [review]
Great work on these updates. The tool is looking/working great.

The only bug I ran into was that I had to capitalize ADCIRC in 2 places in the views.py file. Can you test/confirm this before merging, please? I'll go ahead and approve, but it does seem this edit is needed.

File: Python_env/views.py (hunk 5 above)
Comment (clindeman) [reply Python_env/views.py:24]: I intended for the python file to be lowercase 'adcirc.py'.

File: Python_env/fhitsupport.py (hunk 6 above)
Comment (clindeman) [reply Python_env/fhitsupport.py]: I'll remove the gmail.

File: Python_env/gui_program.py (hunk 7 above)
Comment (clindeman) [reply Python_env/gui_program.py:3]: That is part of a feat that hasn't fully been implemented. The feat is to pull a default unit from a config file for a data source and then allow the user to change it if needed.

File: docs/ADCIRC.txt (hunk 9 above)
Comment (clindeman) [reply docs/ADCIRC.txt]: Yes, there is a method to stream in all the values, each 1000 item chunk at a time. I added some more info to the txt file

File: src/environment.yaml (hunk 10 above)
Comment (clindeman) [reply src/environment.yaml:51]: This tool currently does not use any Hazpy so we could remove it from the environment.yaml

File: Python_env/__init__.py (hunk 12 above)
Comment (clindeman) [reply Python_env/__init__.py]: removed

File: Python_env/TropicalStormNames/readme.md (hunk 13 above)
Comment (clindeman) [reply Python_env/TropicalStormNames/readme.md:1]: Updated

File: Python_env/TropicalStormNames/easternnorthpacific.json (hunk 14 above)
Comment (clindeman) [reply Python_env/TropicalStormNames/easternnorthpacific.json:1]: Yes, they will need to be updated as years go by and if storm names are retired. We will need to manually update them. To automate the process would probably require a custom web scrapper of this site https://www.nhc.noaa.gov/aboutnames.shtml if no other maintained site has an api to get the year, basing and storm names and rules.

**Review by be-cunning — APPROVED** [review]

**Pull request merged**